README.md

```diff
@@ -17,13 +16,13 @@ the square brackets (for multiple-choice questions).
 
 Select all that apply:
 ```
-[] Strings
-[] Booleans
-[] Undefined
+[X] Strings
+[X] Booleans
+[X] Undefined
 [] NaN
-[] Integers
+[X] Integers
 [] Arrays
```

Comment:
...not. :(

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures

Shame on me.